### PR TITLE
Server side rendering

### PR DESCRIPTION
**Pull request by nil1511**
Currently, It is doing server-side rendering for post only. For feeds and users page actions need to changes to be promisable and work with route. For other pages, this can be broken into different PR.

### CODE REVIEW

File: webpack/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+module.exports = require('./makeConfig')({
+  isDevelopmen: true
```
Comment (adcpm) [inline webpack/webpack.config.js]: `isDevelopmen` miss the `t` in the end

**Comment by nil1511** [comment]
@adcpm can you check now? 
Sidebar is not loading because it is not using redux state. 
Server rendering is working only post as of now So we should not remove title and description meta.

**Comment by adcpm** [comment]
I try to share on Facebook and findout it doesnt link to busy but to steemit. Issue is `og:url`, it should always be busy.org, it's not same as canonical.

**Comment by adcpm** [comment]
Also can you add " - Busy" on the end of article title?

**Comment by adcpm** [comment]
Hum. For some reason, sharing on Facebook would not show image & description anymore. 
image
 Strange because that was not the case on last deployment. Something that may cause this is that we have actually 2 <title> tag:
image

**Comment by adcpm** [comment]
Maybe Facebook create image and description based on the URL of og:url.

**Comment by nil1511** [comment]
yes that can be.
should we try with pr app url?

**Comment by adcpm** [comment]
You can make that `og:url` use the current url origin.

**Comment by nil1511** [comment]
Updated. Now it is working.

**Comment by nil1511** [comment]
The method of setting origin now(using global)  is not good. we should eventually switch to env variable in future.

**Pull request merged**